IrinaLoginovaHomeWork_2_2.cpp

```diff
@@ -0,0 +1,38 @@
+#include <cmath>
+#include <cstdint>
+#include <iostream>
+#include <stdexcept>
+
+int main() {
+    int array_NubmerTicket[6];
+    constexpr int MinRange = 1, MaxRange = 9;
+    std::cout << "Please enter 6 digits of your ticket. Numbers ranging from "
+        << MinRange << " to " << MaxRange << std::endl;
+    for (size_t i = 0; i <= 5; ++i) {
```

Comment:
ну не учитывая что  у вас решение с массивами некоректно. Обход массива, а именно условие выхода индекса в цикле делают так
  ```suggestion
    for (size_t i = 0; i < 6; ++i) {
```
это более общий подход